data/json/items/resources/misc.json

```diff
@@ -79,5 +79,31 @@
     "symbol": "=",
     "color": "dark_gray",
     "ammo_type": "components"
+  },
+  {
+    "id": "esapi_plate",
+    "type": "GENERIC",
+    "category": "spare_parts",
+    "name": { "str": "ESAPI ballistic plate" },
+    "description": "An NIJ IV rated ceramic ballistic plate.",
```

Comment:
```suggestion
    "description": "This is a  ceramic plate, polygonal in shape and slightly concave in profile.  No, not the kind for eating out of!  This thick slab of ceramic will prevent penetration by incoming projectiles.  Its inner surface is coated with Ultra High Molecular Weight Polyethylene, and is labelled \"TOP\", while its outer surface is labelled \"STRIKE FACE\".  While you could wear this alone, it will function far better inserted into a ballistic vest.",
```